### DISCUSSION TITLE
Should I see the new core integration?

### DISCUSSION THREAD

**Discussion by zipzagster** [music-assistant/support#3260] [Q&A] [answered]
HAOS. Upgraded to 2024.12.0. Deleted HACS MA integration and restatred. No errors or warnings on restart. The Music Assistant  addon shows connected to HA via the plugin after that plugin was reloaded. The MA server is running.

But from the documentation I expected to see a Music Assistant integration. That integration is neither offered by the system  or available via Add Integration.

**Comment by OzGav** [comment]
I’m not sure how the plugin can be working as that needs the integration…?

**Reply by zipzagster** [reply]
<img width="1465" alt="Screenshot 2024-12-05 at 8 18 03 AM" src="https://github.com/user-attachments/assets/4627e87f-8bb6-445f-85c1-5d2b16f1a75e">
<img width="1466" alt="Screenshot 2024-12-05 at 8 19 02 AM" src="https://github.com/user-attachments/assets/c9cf9afd-d72d-4071-80d7-89fd99d9f77b">

**Reply by OzGav** [reply]
I don’t know how you can’t see it if you click on ADD INTEGRATION  and then search for it 
![IMG_0057](https://github.com/user-attachments/assets/cef35344-0d95-4ef4-8d98-ba514fbda93c)

That image shows the core version first and then the HACS version

**Reply by zipzagster** [reply, marked as answer]
<img width="480" alt="Screenshot 2024-12-05 at 8 33 28 AM" src="https://github.com/user-attachments/assets/3e0016a4-1413-4d7a-a7bd-fd2a4358f1f1">

I'm maybe not on 2024.12.0. I'll figure out how to check that.
